setup.py

```diff
@@ -115,6 +115,13 @@ def _main():
     with open("README.md") as f:
         long_description = f.read()
 
+    # ROCm build of torchaudio needs libomp.so to be packaged with the wheel
+    # to avoid import-time error due to missing dependency
+    if torch.version.hip is not None:
+        import shutil
+
+        shutil.copy("/opt/rocm/llvm/lib/libomp.so", "./torchaudio/lib/")
```

Comment:
What is the test plan for this ?

Reply:
This is a fix for the import of torchaudio using wheel, so in a way, we already have a test case for this, right?